### PR TITLE
Fix bugs related to Firefox

### PR DESCRIPTION
**Pull request by soupette**
My PR is a:

🐛 Bug fix

Main update on the:

Plugin : `content-manager` & `content-type-builder`.

This PR fixes bugs related to Firefox. Relations should be working properly on all navigators and the displayed fields component is also working.

### CODE REVIEW

File: packages/strapi-plugin-content-type-builder/admin/src/components/PopUpRelations/index.js [hunk 1]
```diff
@@ -222,7 +222,8 @@ class PopUpRelations extends React.Component {
   );
 
   renderModalBodyRelations = () => {
-    const header = get(this.props.values, ['params', 'pluginValue'])
+    const pluginValue = get(this.props.values, ['params', 'pluginValue']);
+    const header = !isEmpty(pluginValue) && pluginValue !== ' '
```
Comment (kamalbennani) [inline packages/strapi-plugin-content-type-builder/admin/src/components/PopUpRelations/index.js]: Maybe `!isEmpty(trim(pluginValue))` ?

**Comment by kamalbennani** [comment]
LGTM

**Comment by mnlbox** [comment]
@soupette again thanks for this fix. :pray: You are a hero :wink: 
@lauriejim oh man. I really want to see this fix in 14.4? When you plan for release 14.4?

**Comment by lauriejim** [comment]
Hello @mnlbox , https://github.com/strapi/strapi/projects the release date is not well defined for now.
It will be ~Oct 25, 2018 I think

**Comment by mnlbox** [comment]
@lauriejim Is it merge in 14.4.0??
I can't see it in bug fix part in this link:
https://github.com/strapi/strapi/releases/tag/v3.0.0-alpha.14.4.0

**Comment by lauriejim** [comment]
In bug fix section, the third from the bottom

**Pull request merged**